xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -300,6 +307,22 @@ bool CLinuxRendererGL::Configure(unsigned int width, unsigned int height, unsign
   }
 #endif
 
+  // load 3DLUT
+  if (m_ColorManager->IsEnabled())
+  {
+    if (!m_ColorManager->CheckConfiguration(m_cmsToken, m_iFlags))
+    {
+      CLog::Log(LOGDEBUG, "CMS configuration changed, reload LUT");
+      if (!LoadCLUT())
+        return false;
```

Comment:
@laurimyllari ColorManager needs to be disabled in case or error or the system won't recover.
I enabled ColorManager without a 3dlut file. After this I had to manually edit guisettings to disable colorManager again